protocol/messenger_communities.go

```diff
@@ -924,11 +924,13 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		return nil, err
 	}
 
-	adminsAndTokenMasters := community.GetPrivilegedMembers()
-	for _, communityAdmin := range adminsAndTokenMasters {
-		_, err := m.sender.SendPrivate(context.Background(), communityAdmin, &rawMessage)
-		if err != nil {
-			return nil, err
+	if !community.AcceptRequestToJoinAutomatically() {
+		adminsAndTokenMasters := community.GetPrivilegedMembers()
+		for _, communityAdmin := range adminsAndTokenMasters {
+			_, err := m.sender.SendPrivate(context.Background(), communityAdmin, &rawMessage)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Here you do not send requests to join as a sender, I think we need also to filter out messages on the HandleRequestToJoin side